tests/test_monkeypatch.py

```diff
@@ -92,7 +92,7 @@ def test_get_archive_link(self, _mock):
             archive_link = ""
         # We test that moderation check is called when getting an edit link
         self.assertEqual(1, _mock.call_count)
-        if versioning_version < "2":
+        if versioning_version != "2.0.0":
             # Edit link is inactive as `mock_is_obj_review_locked` is True
             self.assertIn("inactive", archive_link)
         else:
```

Comment:
```suggestion
        self.assertIn("inactive", archive_link)
```
**plus** deleting the next three lines